app/models/cart.rb

```diff
@@ -17,6 +17,11 @@ def add_freelancer(id)
     contents[id.to_s] = (contents[id.to_s] || 0) + 1
   end
 
+  def delete_freelancer(id)
+    contents[id.to_s] = contents[id.to_s] - 1
```

Comment:
after rereading this story, i don't think this -1 is necessary. 

we would have ```contents.delete(id.to_s)``` that would remove the item entirely from the cart.

Reply:
great- making the update

Reply:
please see updated commit